### PR TITLE
feat: create CODEOWNERS

### PR DESCRIPTION
**Pull request by SgtPooki**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence
+# these owners will be requested for review when someone
+# opens a pull request.
+# All GUI Teams: @ipfs-shipyard/ipfs-gui @ipfs-shipyard/gui @ipfs/gui-dev @ipfs/wg-gui-ux
+*   @ipfs/gui-dev
```
Comment (lidel) [inline .github/CODEOWNERS:8]: Should we limit this to the actual website? 


```suggestion
site   @ipfs/gui-dev
```

**Review by lidel — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (SgtPooki) [reply .github/CODEOWNERS]: I commit-ed this suggestion but i'm not sure I understand what you mean by "without the low level build CI with signing"

**Comment by SgtPooki** [comment]
@lidel should we have a default code-owner? `* @someone` ?

**Review by lidel — APPROVED** [review]
No, usually people familiar with release review / make the merge call.
\+  we triage this repo as part of Go/Kubo triage already,

**Pull request merged**